jwql/website/apps/jwql/views.py

```diff
@@ -473,16 +476,32 @@ def view_image(request, user, inst, file_root, rewrite=False):
     HttpResponse object
         Outgoing response sent to the webpage
     """
+
     # Ensure the instrument is correctly capitalized
     inst = JWST_INSTRUMENT_NAMES_MIXEDCASE[inst.lower()]
 
     template = 'view_image.html'
     image_info = get_image_info(file_root, rewrite)
+
+    # Determine current flagged anomalies
+    current_anomalies = get_current_flagged_anomalies(file_root)
+
+    # Create a form instance
+    form = AnomalySubmitForm(request.POST or None, initial={'anomaly_choices': current_anomalies})
+
+    # If this is a POST request, process the form data
+    if request.method == 'POST':
+        anomaly_choices = dict(request.POST)['anomaly_choices']
```

Comment:
Is this `dict(request.POST)` line just basically accessing the `AnomalySubmitForm` attributes? Or where is the `anomaly_choices` key coming from?

Reply:
@laurenmarietta Yes, exactly.  I'm not sure if there is a better way to do it?

Reply:
I certainly don't know of one - I just didn't know you could extract info from `request.POST` like this. Neat!